packages/pn-pa-webapp/src/navigation/routes.const.ts

```diff
@@ -1,5 +1,6 @@
 
 export const DASHBOARD = '/dashboard';
+export const DETTAGLIO_NOTIFICA = '/dashboard/:id/notifica';
```

Comment:
@AndreaCimini90 questa scrivila come DASHBOARD+'/:id/' + NOTIFICA

Reply:
@AndreaCimini90 anzi meglio: 
trasformiamola in una funzione al quale possiamo passare l':id come param tipo:

DETTAGLIO_NOTIFICA(id:string) return DASHBOARD+ id + NOTIFICA